examples/falqon/main.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+from qibo import models, hamiltonians
+
+def main(nqubits, delta_t=.1, max_layers=100):
+    # create XXZ Hamiltonian for nqubits qubits
+    hamiltonian = hamiltonians.XXZ(nqubits)
+    # create FALQON model for this Hamiltonian
+    falqon = models.FALQON(hamiltonian)
+
+    best_energy, final_parameters = falqon.minimize(delta_t, max_layers)[:2]
+
+    print('The optimal energy found is', best_energy)
+
+    return best_energy, final_parameters
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--nqubits", default=5, type=int)
+    parser.add_argument("--delta_t", default=.1, type=float)
+    parser.add_argument("--max_layers", default=100, type=int)
+    args = vars(parser.parse_args())
+    main(**args)
```

Comment:
Please add a new line at the end of the file.

Reply:
@AdrianPerezSalinas if you use vscode, you can set this automatically when saving the file, and for the PEP8 specs (like 2 empty lines) you can do `crtl+shift i` and this will align the code.